### ISSUE TITLE
Issue with Word Paste plugin using MS Edge

### ISSUE THREAD

**Issue by dswitzer** [froala/wysiwyg-editor#1812]
The Word Paste plugin isn't working correctly for me in MS Edge. The problem appears to be that in MS Edge, the results from DOMParser.parseFromString() does not appear to return the `children` property.

I was able to fix the issue by changing the line:

```
var html_node = word_doc.children[0];
```

To:

```
var html_node = word_doc.firstChild;
```
Without making this change, any time the Word Paste plugin is triggered with MS Edge, an error would occur.

**Comment by dswitzer** [comment]
I can replicate the issue just by running the following in the console:

```
(new DOMParser()).parseFromString("<h1>hello world</h1>", 'text/html').children
```

If I run this in Edge, I get undefined. Changing `children` to `firstChild` gets me what I expect.

The use of `firstChild` works in all supported browser.

**Comment by stefanneculai** [comment]
It's fixed on the current master.

**Comment by dswitzer** [comment]
Great work! This is fixed in the new 2.5.1 release. Thanks!
